src/components/TextField/TextField.tsx

```diff
@@ -278,37 +280,89 @@ export class TextField extends React.Component<ITextFieldProps, ITextFieldState>
+                    onFocus={this._onFocus}
+                    onBlur={this._onBlur}
+                />
+            );
+        }
+
         return (
-            <textarea
-                { ...textAreaProps }
-                id={this._id}
-                ref={(c): HTMLTextAreaElement => this._field = c}
-                value={this.state.value}
-                onChange={this._onInputChange}
-                onKeyUp={this._onKeyUp}
-                className={this._fieldClassName}
-                onFocus={this._onFocus}
-                onBlur={this._onBlur}
-            />
+            <Tooltip
+                content={this.props.errorMessage}
+                className="tooltip-error"
+                showTooltip={this.state.isFocused}
+                directionalHint={DirectionalHint.bottomLeftEdge}>
+                <div>
+                    <textarea
+                        { ...textAreaProps }
+                        id={this._id}
+                        ref={(c): HTMLTextAreaElement => this._field = c}
+                        value={this.state.value}
+                        onChange={this._onInputChange}
+                        onKeyUp={this._onKeyUp}
+                        className={this._fieldClassName}
+                        onFocus={this._onFocus}
+                        onBlur={this._onBlur}
+                    />
+                </div>
+            </Tooltip>
         );
     }
 
     private _renderInput(): React.ReactElement<React.HTMLProps<HTMLInputElement>> {
         let inputProps = getNativeAttributes<React.HTMLProps<HTMLInputElement>>(this.props, inputAttributes, ['defaultValue']);
 
+        if (!this.props.errorMessage) {
+            return (
+                <input
+                    type={'text'}
+                    { ...inputProps }
+                    id={this._id}
+                    ref={(c): HTMLInputElement => this._field = c}
+                    value={this.state.value}
+                    onChange={this._onInputChange}
+                    onKeyUp={this._onKeyUp}
+                    className={this._fieldClassName}
+                    onFocus={this._onFocus}
+                    onBlur={this._onBlur}
+                />
+            );
+        }
+
         return (
-            <input
-                type={'text'}
-                { ...inputProps }
-                id={this._id}
-                ref={(c): HTMLInputElement => this._field = c}
-                value={this.state.value}
-                onChange={this._onInputChange}
-                onKeyUp={this._onKeyUp}
-                className={this._fieldClassName}
-                onFocus={this._onFocus}
-                onBlur={this._onBlur}
-            />
+            <Tooltip
+                content={this.props.errorMessage}
+                className="tooltip-error"
+                showTooltip={this.state.isFocused}
+                directionalHint={DirectionalHint.bottomLeftEdge}>
+                <div>
```

Comment:
I feel like this div should have assignable className so it can be styled if there is need?